external/Microsoft.SqlTools.Hosting.v2/Utility/CommandOptions.cs

```diff
@@ -144,12 +144,22 @@ public virtual void SetLocale(string locale)
         /// culture-specific messages
         /// </summary>
         /// <param name="locale"></param>
-        internal void LocaleSetter(string locale)
+        internal void LocaleSetter(string locale, Boolean useUserSettings = false)
```

Comment:
What is the default for? Is there a scenario where the default would be used?

Reply:
Likely not, but I put this in just in case if it won't break anything, I'll remove it